content/docs/notebooks/why-use-jupyter-notebook.md

```diff
@@ -0,0 +1,16 @@
++++
+title = "Overview: Using Jupyter Notebooks in Kubeflow"
+description = "Why use Jupyter Notebooks in Kubeflow"
+weight = 5
++++
+
+
+The benefit of integrating Jupyter Notebooks service in Kubeflow is that:
```

Comment:
Suggested change for line 8:

"There are multiple benefits to integrating Jupyter notebooks in Kubeflow for enterprise environments. The benefits include:"